x-pack/plugins/watcher/public/sections/watch_detail/components/watch_detail/watch_action_status.tsx

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
@pcsanwald I think this is something that I'd like to reuse for the advanced watcher pages too. Maybe we should move it under `/watcher/public/components`?

Generally speaking, I'm not sure how we want to approach the file structure. However, right now, it seems we have some common components in there already (e.g., `form_errors.tsx`, `delete_watches_modal.tsx`) FYI @bmcconaghy

Reply:
yeah seems like a good candidate for moving up to the common components dir.

Reply:
should we do this after merge, or should I go ahead and do in this PR? I'm fine either way

Reply:
another PR is fine I think